### PR TITLE
Teams/Contexts Section Edits

### PR DESCRIPTION
**Pull request by jdinan**
* Clean up and clarify text
* Found and fixed several errors
* Move context options to contexts intro since they are used by both creation routines

### CODE REVIEW

**Comment by jdinan** [comment]
@agrippa @wrrobin @naveen-rn Please review these edits.

File: content/shmem_team_split_2d.tex [hunk 1]
```diff
@@ -38,22 +38,24 @@
...
-new teams by splitting an existing parent team into up to two subsets based on a
+The \FUNC{shmem\_team\_split\_2d} routine is a collective operation. It returns two
+new teams to the calling \ac{PE} by splitting an existing parent team into subsets based on a
 2D Cartesian space. The user provides the size of the \VAR{x} dimension, which is then
 used to derive the size of the \VAR{y} dimension based on the size of the parent team.
 The size of the \VAR{y} dimension will be equal to $\lceil N \div xrange \rceil$, where
 \VAR{N} is the size of the parent team. In other words,
 $xrange \times yrange \geq N$, so that every \ac{PE} in the parent team has a
-unique \VAR{(x,y)} location the 2D Cartesian space.
+unique \VAR{(x,y)} location the 2D Cartesian space. The resulting \VAR{xaxis\_team}
```
Comment (agrippa) [inline content/shmem_team_split_2d.tex]: grammar nit: should it be "location in the 2D Cartesian space"?

File: content/shmem_team_split_strided.tex [hunk 2]
```diff
@@ -30,40 +30,40 @@
...
+  \;
   \forall
-  \hspace{0.35em}
-  i \in \mathbb{Z}_{size}
+  \;
+  i \in \mathbb{Z}_{size-1}
 \end{equation*}
-where $N$ is the number of \acp{PE} in the parent team and $size$ is greater than zero.
+where $\mathbb{Z}$ is the set of natural numbers ($0, 1, \dots$), $N$ is the
+number of \acp{PE} in the parent team and $size$ is a positivie number indicating
```
Comment (agrippa) [inline content/shmem_team_split_strided.tex]: typo: positivie -> positive

**Review by agrippa — CHANGES_REQUESTED** [review]

**Comment by jdinan** [comment]
@agrippa Thanks for the edits. Applied in fbdaeb6.

**Review by agrippa — APPROVED** [review]

File: content/shmem_team_split_2d.tex [hunk 3]
```diff
@@ -38,22 +38,24 @@
...
 \apiargument{OUT}{yaxis\_team}{A new \ac{PE} team handle representing a \ac{PE}
-subset consisting of all the \acp{PE} that have the same coordinate along the \VAR{y}-axis
+subset consisting of all the \acp{PE} that have the same coordinate along the \VAR{x}-axis
 as the calling \ac{PE}.}
 \end{apiarguments}
 
 \apidescription{
-The \FUNC{shmem\_team\_split\_2d} routine is a collective routine. It creates two
-new teams by splitting an existing parent team into up to two subsets based on a
+The \FUNC{shmem\_team\_split\_2d} routine is a collective operation. It returns two
```
Comment (wrrobin) [inline content/shmem_team_split_2d.tex:46]: Should we say `routine **performs** a collective operation`?

File: content/teams_intro.tex [hunk 4]
```diff
@@ -87,13 +89,12 @@ \subsubsection*{Team Creation}
 contexts (see Section~\ref{sec:ctx}) or using the translation routine
 \FUNC{shmem\_team\_translate\_pe}.
 
-As with any collective routine on a team, the program must ensure that there
+As with any collective routine on a team, the program must ensure that in multithreaded executions there
```
Comment (wrrobin) [inline content/teams_intro.tex]: executions**,**

File: content/shmem_team_split_2d.tex (hunk 3 above)
Comment (jdinan) [reply content/shmem_team_split_2d.tex:46]: The phrasing "is a collective" appears in 11 separate API descriptions and I found no instances of "performs a collective". Your phrasing is a bit better, but I think we should stick with consistency for now.

File: content/teams_intro.tex (hunk 4 above)
Comment (jdinan) [reply content/teams_intro.tex]: @wrrobin I've rewritten this sentence in d65b72d to reference the full description of team-based collective requirements and to also hopefully make it a little easier to digest. Please let me know what you think.

Comment (wrrobin) [reply content/teams_intro.tex]: @jdinan The updated text looks good. Thanks.

File: content/shmem_team_split_2d.tex (hunk 3 above)
Comment (wrrobin) [reply content/shmem_team_split_2d.tex:46]: We are not introducing the definitions for `operation` and `routine` this time, so I agree that being consistent would be the better option.

**Review by wrrobin — APPROVED** [review]

**Comment by jdinan** [comment]
@naveen-rn Could you please review?

**Comment by jdinan** [comment]
Going ahead with merge. @naveen-rn Please review when you have a chance.

**Pull request merged**